src/Control/Carrier/Pure.hs-boot

```diff
@@ -0,0 +1,7 @@
+module Control.Carrier.Pure
```

Comment:
I don’t love that we had to add this, but it keeps the change smaller than the other options, and preserves the useful property that an exported `run` from a given carrier means that you can do useful work with a single import.

Reply:
Yeah, I think this is fine 👍